droidlet/lowlevel/hello_robot/remote/README.md

```diff
@@ -20,6 +20,29 @@ popd
 
 ## 3. Start the services
 
+### Without ROS (i.e. directly using base API)
+
 ```bash
+# export LOCOBOT_IP=[your network ip]
+# example when using tailscale:
+# export LOCOBOT_IP=$(tailscale ip --4)
+
 ./launch.sh
+```
+
+### With ROS (i.e. better odometry using hector_slam, etc.)
+
+```bash
+# in fresh terminal
+deactivate
+./launch_ros.sh
```

Comment:
extra steps for launching ROS - should we discuss? I think if its only one ROS launch file maybe it could go into submodule or MRP 

doesnt need to be a change for this PR though

Reply:
Will move everything to MRP in an upcoming PR